webapp/channels/src/actions/global_actions.tsx

```diff
@@ -374,8 +374,9 @@ export async function redirectUserToDefaultTeam() {
     const teamId = LocalStorageStore.getPreviousTeamId(user.id);
 
     let myTeams = getMyTeams(state);
-    if (myTeams.length === 0) {
-        if (isUserFirstAdmin) {
+    const teams = getActiveTeamsList(state);
+    if (myTeams.length === 0 || teams.length === 0) {
```

Comment:
Given that this is the first admin scenario, we probably don't need to check for `myTeams` any longer here